src/backend/base/langflow/components/brightdata/brightdata_structured_data.py

```
@@ -0,0 +1,1112 @@
+        # Sort by total score
+        all_scores.sort(key=lambda x: x["total_score"], reverse=True)
+        
+        detection_details = {
+            "url": url,
+            "parsed_domain": domain,
+            "total_datasets_checked": len(datasets),
+            "datasets_with_scores": len(all_scores),
+            "all_scores": all_scores[:10],  # Top 10 for brevity
+            "detection_method": "enhanced_ai_scoring"
+        }
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)